parts/masterparams.t

```diff
@@ -52,6 +52,13 @@
         "description": "A string hash of the master DNS name to uniquely identify the cluster."
       },
       "type": "string"
+    },
+    "targetEnvironment": {
+      "defaultValue": "AzurePublicCloud",
```

Comment:
same as my other comment, these should be AzureCloud, AzureChinaCloud, AzureUSGovernment, and AzureGermanCloud.